src/agoradatatools/etl/utils.py

```diff
@@ -110,7 +110,6 @@ def rename_columns(df: pd.DataFrame, column_map: dict) -> pd.DataFrame:
         df.rename(columns=column_map, inplace=True)
     except TypeError:
         print("Column mapping must be a dictionary")
-        return df
 
```

Comment:
I deleted this "return df" statement because SonarCloud complained about it: the function always returns df whether there's an error or not.